sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -591,3 +591,10 @@
   font-family: "BCSans-bold", "BCSans", "Noto Sans", Verdana, Arial, sans-serif;
   color: $brand-blue;
 }
+
+.restrictions-details {
+  font-style: normal;
+  font-weight: 400;
```

Comment:
As @ann-aot mentioned you can use the font-size variables and for font-weight we have $font-weight-normal variable.

Reply:
ok done.